### PR TITLE
Review of PTNFLY-938 Adding Find to the toolbar #383

### PR DESCRIPTION
**Pull request by cardosogabriel**
## Description

Code review for PTNFLY-938 Adding Find to the toolbar #383
- Fixed bug on IE (text cut and elements overriden, see image below).
- Rewrote toolbar-pf-view-selector componente using buttons instead of a list of links
- Removed unnecessary attributes that either has no effect or were already declared by other piece of code
- Put styles with one line attribute in one line.
## Link to rawgit and/or image

[rawgit link](http://rawgit.com/cardosogabriel/patternfly/review-pr-393-dist/dist/tests/card-view-single-select.html)

IE 10 bug

### CODE REVIEW

File: less/toolbar.less [hunk 1]
```diff
@@ -58,42 +51,21 @@
...
-    }
-  }
 }
 .toolbar-pf-find {
   font-size: (@font-size-base + 2);
   position: relative;
-  .btn {
-    outline: none;
-  }
+  .btn { outline: none; }
```
Comment (andresgalante) [inline less/toolbar.less]: @cardosogabriel Please lets leave the outline.

Comment (cardosogabriel) [reply less/toolbar.less]: Removed this declaration.

**Comment by andresgalante** [comment]
👍 Thanks @cardosogabriel

**Comment by jeff-phillips-18** [comment]
When clicking on the search button, it gets a big blue focus border. Is that desirable?

**Comment by andresgalante** [comment]
Forms and btns should have a focus, like this:
https://www.patternfly.org/pattern-library/widgets/#forms
https://www.patternfly.org/pattern-library/widgets/#buttons

@jeff-phillips-18 if that is what you get, then yes, that's expected.

**Comment by jeff-phillips-18** [comment]
OK then. LGTM

**Comment by andresgalante** [comment]
@jeff-phillips-18 thanks, lets merge it then 👍 

@cardosogabriel thanks!

**Comment by cardosogabriel** [comment]
Hi @junezhang . Which margin-bottom you are talking about? Can u point me to the line? Thanks

File: less/toolbar.less [hunk 2]
```diff
@@ -39,15 +35,12 @@
...
-        color: @link-color;
-      }
+      &:hover { color: @link-color; }
     }
   }
 }
 .toolbar-pf-actions {
   display: table;
-  margin-bottom: 10px;
   width: 100%;
```
Comment (junezhang) [inline less/toolbar.less:44]: @cardosogabriel  Here, this margin-bottom. :-)
Please reference #383 the sixth comments from LHinson.

**Comment by junezhang** [comment]
@cardosogabriel, Leave the comment in the line. Please have a check. Thanks.

**Comment by cardosogabriel** [comment]
Thanks @junezhang . Will open a new PR to include this margin-bottom.

**Pull request merged**